spec/system/meta_tags_spec.rb

```diff
@@ -0,0 +1,45 @@
+RSpec.describe "Meta Tags" do
```

Comment:
Well done for spotting these tests and bringing them over! 🎉 

I think it would really useful to have these tests as a shared example as all of the document types will need to render meta tags and they're an easy thing to miss. 

Perhaps convert them to shared examples where the content item and route being tested can be passed in?

Reply:
Now shared.